skf/markdown/code_examples/java/31-code_example--Single_input_validation_control--.md

```diff
@@ -0,0 +1,131 @@
+Single input validation control 
+--------------------------------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class inputValidation {
```

Comment:
Could you change the class to InputValidation and update the other places, thanks.